python/sglang/srt/managers/schedule_batch.py

```diff
@@ -1132,7 +1136,7 @@ def prepare_for_extend(self):
         self.extend_input_logprob_token_ids = extend_input_logprob_token_ids
 
         # Write to req_to_token_pool
-        if global_server_args_dict["attention_backend"] != "torch_native":
+        if global_server_args_dict["attention_backend"] not in ["torch_native", "hpu"]:
```

Comment:
this name of attention_backend is really confusing

Reply:
I changed it to "hpu_attn_backend". Do you think it is more clear, or do you have other suggestions?